db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```

Comment:
Kindly include a seed file to help create greetings for your application when a reviewer is testing.

You can do this by using `Greeting.create!(greeting: "Hi")` and other greetings